src/subcommand/server.rs

```diff
@@ -295,7 +309,6 @@ impl Server {
             .allow_origin(Any),
         )
         .layer(CompressionLayer::new())
-        .layer(TimeoutLayer::new(Duration::from_secs(self.timeout.unwrap_or(30))))
```

Comment:
curious: why didn't this work?

Reply:
not totally sure, I suspect that the very slow requests we saw were partially due to no concurrency limit, so there were just too many threads spawned and the timeout didn't trigger at the expected time

Reply:
I was able to see timeout (408 status) locally when picking a very low timeout, so it is definitely configured correctly